packages/ui-buttons/src/BaseButton/index.tsx

```diff
@@ -356,38 +355,21 @@ class BaseButton extends Component<Props> {
     )
 
     const flexChildren = hasOnlyIconVisible ? (
-      <Flex.Item>
+      <span css={styles.flexOnlyIcon}>
```

Comment:
Please consider the renaming of the classes I sent you on slack.

Reply:
done